program/program/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+
+namespace ConsoleApp
+{
+    class Library
+    {
+        public string Name;
+    }
+
+    class Visitors
+    {
+        private int BirthDay { get; init; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+
+    }
+
+    class VisitorsTiket
+    {
+        private string Name { get; set; }
+        private int DateWhenTakeTheBook { get; set; }
+        public int NameOfTheBook { get; set; }
+        private int DateOfValidationTiket { get; set; }
+        private object Stamp { get; init; }
+        private object Sign { get; init; }
+
+    }
+
+    class Archive
+    {
+        public string NameOfBooks { get; init; }
+        public string AuthorOfBooks { get; init; }
```

Comment:
I think it is not needed, you have author in book class, you can reference book instead